### PR TITLE
[zh] sync concepts/configuration/manage-resources-containers

### PR DESCRIPTION
**Pull request by KKtheGhost**
## PR Summary

- **EN upstream**: `content/en/docs/concepts/configuration/manage-resources-containers.md`
- **Sync to**: `content/zh-cn/docs/concepts/configuration/manage-resources-containers.md`
- **Updates**:
  - Sync'ed a new `NOTE` to manage-resources-containers.md from the EN upstream.

Sync check by`scripts/lsync.sh` on `main` branch:
```diff
$ scripts/lsync.sh content/zh-cn/docs/concepts/configuration/manage-resources-containers.md
diff --git a/content/en/docs/concepts/configuration/manage-resources-containers.md b/content/en/docs/concepts/configuration/manage-resources-containers.md
index 45686178d6..f5ec08df70 100644
--- a/content/en/docs/concepts/configuration/manage-resources-containers.md
+++ b/content/en/docs/concepts/configuration/manage-resources-containers.md
@@ -256,6 +256,18 @@ Your applications cannot expect any performance SLAs (disk IOPS for example)
 from local ephemeral storage.
 {{< /caution >}}

+
+{{< note >}}
+To make the resource quota work on ephemeral-storage, two things need to be done:
+
+* An admin sets the resource quota for ephemeral-storage in a namespace.
+* A user needs to specify limits for the ephemeral-storage resource in the Pod spec.
+
+If the user doesn't specify the ephemeral-storage resource limit in the Pod spec,
+the resource quota is not enforced on ephemeral-storage.
+
+{{< /note >}}
+
 Kubernetes lets you track, reserve and limit the amount
 of ephemeral local storage a Pod can consume.

```
Sync check by`scripts/lsync.sh` on `sync/configuration/manage-resources-containers` branch:
```diff
$ scripts/lsync.sh content/zh-cn/docs/concepts/configuration/manage-resources-containers.md
content/zh-cn/docs/concepts/configuration/manage-resources-containers.md is still in sync
```

Thanks to the reviewers in advance.
Best Regards,

Kivinsae Fang

### CODE REVIEW

File: content/zh-cn/docs/concepts/configuration/manage-resources-containers.md [hunk 1]
```diff
@@ -469,6 +478,17 @@ kubelet 也使用此类存储来保存[节点层面的容器日志](/zh-cn/docs/
...
+{{< note >}}
+为了使临时性存储的资源配额生效，需要完成以下两个步骤：
+
+* 管理员在命名空间中设置临时性存储的资源配额。
+* 用户需要在 Pod spec 中指定临时性存储资源的限制。
+
+如果用户在 Pod spec 中未指定临时性存储资源的限制，
+则临时性存储的资源配额不会生效。
+
+{{< /note >}}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/configuration/manage-resources-containers.md:490]: 通常做成这种对照形式：

```
{{< note >}}
<!--
This is original english.
-->
这是英语原文。
{{< /note >}}
```

**Review by windsonsea — COMMENTED** [review]
请参阅 [中文本地化风格指南 - 短代码（shortcode）处理](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#shortcode)

**Comment by KKtheGhost** [comment]
@windsonsea 
Thanks so much for the review and suggestion.
PR was updated and squashed according to the doc conventions in [中文本地化风格指南 - 短代码（shortcode）处理](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#shortcode)

**Comment by windsonsea** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**